vector/src/main/java/im/vector/services/EventStreamService.java

```diff
@@ -353,7 +354,7 @@ public int onStartCommand(Intent intent, int flags, int startId) {
                     return START_NOT_STICKY;
                 }
 
-                if ((null != VectorApp.getInstance()) && VectorApp.getInstance().didAppCrash()) {
+                if (VectorUncaughtExceptionHandler.INSTANCE.didAppCrash(this)) {
```

Comment:
Does that mean that notifications won't work until the app is manually restarted?

Reply:
I assume yes, according to git history, it was added to fix https://github.com/vector-im/riot-android/issues/1262

Reply:
Maybe we should display a notification in this case? 'Oups, sorry but Riot did crash. In order to maintain your system integrity Riot will not restart background activities until you re-start the app manually, this might affect notification reliability'

Reply:
Yes, maybe, but I think we can do this in a dedicated PR